tests/Unit/Helpers/PointwiseFunctions/AnalyticSolutions/FirstOrderEllipticSolutionsTestHelpers.hpp

```diff
@@ -221,11 +221,13 @@ void verify_smooth_solution(
     const double tolerance_offset, const double tolerance_scaling,
     PackageFluxesArgs&& package_fluxes_args) {
   INFO("Verify smooth solution");
+  const size_t max_points =
+      (Spectral::maximum_number_of_points<Spectral::Basis::Legendre> >= 12)
```

Comment:
I think this is just `std::min(Spectral::maximum_number_of_points<Spectral::Basis::Legendre>, 12)` (maybe `12_st`), similar elsewhere